openfeature/transaction_context/transaction_context_propagator.py

```diff
@@ -0,0 +1,14 @@
+import typing
+from typing import TypeVar
+
+from openfeature.evaluation_context import EvaluationContext
+
+T = TypeVar("T", bound="TransactionContextPropagator")
```

Comment:
Maybe you planed to somehow use it, but I don't see anything. please elaborate or remove it.

Reply:
Yes it was from before, thanks!